### PR TITLE
[DMN (yet)] [NP-641 Update] Keep next enabled

### PR DESCRIPTION
**Pull request by KernelDeimos**
This PR keeps the wizard's next button enabled

### CODE REVIEW

**Comment by Nauna** [comment]
We are assuming this is a quick fix till we identify possible functionality - given the option to not check anything on 'next' or to check partial validation on 'next'.

realized that we can't check full validation - given the discussion surrounding the 'reviewed' prop explicit to SigningOfficers.

I approve this.

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/crunch/ui/CapabilityWizardlet.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ foam.CLASS({
           return this;
         });
       }
+    },
+    {
+      name: 'readyToSubmit',
```
Comment (jlhughes) [inline src/foam/nanos/crunch/ui/CapabilityWizardlet.js]: This was renamed to mustBeValid on the foam side was it not?

Comment (KernelDeimos) [reply src/foam/nanos/crunch/ui/CapabilityWizardlet.js]: Oops, you're right - will update

Comment (KernelDeimos) [reply src/foam/nanos/crunch/ui/CapabilityWizardlet.js]: Adding DNM tag - need to check something

**Comment by KernelDeimos** [comment]
This isn't needed anymore since `mustBeValid` defaults to false

**Pull request closed without merging**